### ISSUE TITLE
Inconsistent web source in Linux portable builds for 10.5.2

### ISSUE THREAD

**Issue by R-Bas** [jellyfin/jellyfin#2648]
Getting this error when first accessing jellyfin through a web browser after following the "Linux (generic amd64)" installation process from the [documentation](https://jellyfin.org/docs/general/administration/installing.html) :

[08:53:31] [ERR] [24] Emby.Server.Implementations.HttpServer.HttpListenerHost: Error processing request: Could not find file '.../jellyfin/jellyfin_10.5.2/jellyfin-web/componentscastSenderApi.js'.

It seems to only be a typing error (/ is missing) as renaming the file '.../jellyfin/jellyfin_10.5.2/jellyfin-web/components/castSenderApi.js' to '.../jellyfin/jellyfin_10.5.2/jellyfin-web/componentscastSenderApi.js' fixes the issue.

**Comment by JettScythe** [comment]
I also have the same bug, though it only affects my Brave Browser for some reason.
Works with firefox just fine.

**Comment by joshuaboniface** [comment]
So it looks like yet again builds were inconsistent, somehow that I can't figure out, this particular one built with Master web instead of the Release web. This has become a major focus going forward.

For now, if you're using the portable Linux build, you can fix this yourself by doing the following somewhere:

```
git clone https://github.com/jellyfin/jellyfin-web
pushd jellyfin-web/
git checkout tags/v10.5.2
npx yarn install
sudo mv dist/* /path/to/your/Jellyfin/install/directory/jellyfin-web/
```

That will manually build 10.5.2 web instead of master web and put it into your install.

**Comment by JettScythe** [comment]
> So it looks like yet again builds were inconsistent, somehow that I can't figure out, this particular one built with Master web instead of the Release web. This has become a major focus going forward.
> 
> For now, if you're using the portable Linux build, you can fix this yourself by doing the following somewhere:
> 
> ```
> git clone https://github.com/jellyfin/jellyfin-web
> pushd jellyfin-web/
> git checkout tags/v10.5.2
> npx yarn install
> sudo mv dist/* /path/to/your/Jellyfin/install/directory/jellyfin-web/
> popd
> ```
> 
> That will manually build 10.5.2 web instead of master web and put it into your install.

Perfect fix for me! Thank you!

**Comment by DieselTech** [comment]
This doesn't just effect the linux portable build. It's present in the jellyfin_10.5.2.portable_windows-x64 also.

**Comment by Tea-NT** [comment]
> 这不仅会影响linux的可移植版本。它也存在于jellyfin_10.5.2.portable_windows-x64中。

+1

**Comment by Berghopper** [comment]
+1 for the slash missing.

I actually tried hotfixing this by making a symbolic links towards the broken resource;
`ln -s  /path/to/install/jellyfish-web/components/castSenderApi.js /path/to/install/jellyfish-web/componentscastSenderApi.js`

However when doing this, I just got more issues;

```
ERR] [24] Emby.Server.Implementations.HttpServer.HttpListenerHost: Error processing request
System.InvalidOperationException: Response Content-Length mismatch: too many bytes written (582 of 63).
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.ThrowTooManyBytesWritten(Int32 count)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.VerifyAndUpdateWrite(Int32 count)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.FirstWriteAsyncInternal(ReadOnlyMemory`1 data, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.FirstWriteAsync(ReadOnlyMemory`1 data, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.WritePipeAsync(ReadOnlyMemory`1 data, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpResponseStream.WriteAsync(Byte[] buffer, Int32 offset, Int32 count, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.ResponseCompression.ResponseCompressionBody.WriteAsync(Byte[] buffer, Int32 offset, Int32 count, CancellationToken cancellationToken)
   at System.IO.Stream.CopyToAsyncInternal(Stream destination, Int32 bufferSize, CancellationToken cancellationToken)
   at Emby.Server.Implementations.HttpServer.FileWriter.TransmitFile(Stream stream, String path, Int64 offset, Int64 count, FileShare fileShare, CancellationToken cancellationToken)
   at Emby.Server.Implementations.HttpServer.FileWriter.WriteToAsync(HttpResponse response, CancellationToken cancellationToken)
   at Emby.Server.Implementations.Services.ServiceHandler.ProcessRequestAsync(HttpListenerHost httpHost, IRequest httpReq, HttpResponse httpRes, ILogger logger, CancellationToken cancellationToken)
   at Emby.Server.Implementations.HttpServer.HttpListenerHost.RequestHandler(IHttpRequest httpReq, String urlString, String host, String localPath, CancellationToken cancellationToken)
```

This seems to only happen in chrome, every other browser seems to work just fine.

**Comment by matt-derrick** [comment]
I don't know about other browsers, but I was getting this error as well in brave browser until i turned off their 'sheilds' addon for jellyfin (localhost:8096). Then it started working again.

**Comment by rodrigodemingo** [comment]
I have the same issue in Chrome 80 (only working in Safari).

**Comment by joshuaboniface** [comment]
This should be fixed with 10.5.3 - I've gone through all the build configs and spot-checked a few specific ones and they were correct now. If there are any further issues with this please reply here.

**Comment by rodrigodemingo** [comment]
Is it possible to have a .dmg?
